e2e/language/testdata/python/conda_channel/build.envd

```diff
@@ -0,0 +1,10 @@
+def build():
+    config.conda_channel(
+        channel="""
+channels:
+  - defaults
+  - conda-forge # dgllife in env.yaml only works if specify channels here
```

Comment:
https://github.com/tensorchord/envd/blob/d87afda9213fcb334ca55d247ba3c0663b7b180b/pkg/lang/ir/install-conda.sh#L24
Already use `conda-forge`. So not sure if this test case can fail.

Reply:
I think it's fine. The condarc in install-conda.sh is used to install python. And the conda_channel is for user configuration, I think it's fine to overwrite?

Reply:
It's okay to override. But this won't be able to detect if the new channel config is right or wrong. Because even if it doesn't work, the default channel is already `conda-froge`, and the test always passes.

Reply:
Changed to use dgl as test target

Reply:
Changed to use dgl which needs `dglteam` channel for testing